Makefile

```
@@ -1,11 +1,19 @@
+.PHONY: check-format
+check-format:
+	black  --check --config black.toml .
+
 .PHONY: format
 format:
-	black  --check --config black.toml .
+	black  --config black.toml .
 
-.PHONY: isort
-imports:
+.PHONY: check-isort
+check-imports:
```

```suggestion
.PHONY: check-imports
check-imports:
```
Noi pitää matchaa.
Toi `.PHONY: jotain` merkkaa targetin `jotain:` sellaseks et se suoritetaan aina. Make ei normisti suorita targettia jos sillä nimellä on jo olemassa tiedosto joka on luotu sen riippuvuuksien jälkeen.

```console
$ cat Makefile
target:
        @echo "Jotain"
$ make target
Jotain
$ touch target
$ make target
make: 'target' is up to date.
$ echo ".PHONY: target" >> Makefile
$ make target
Jotain
```

Jotain Makefile juttuja :D
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

---

Aijjaa ku sul oli
Phony isort
Imports:

Nii luulin et se oli tarkoituksella

---

Oho, sori oliki mun moka :D